src/coffee/services/subscriptions.coffee

```diff
@@ -0,0 +1,12 @@
+BaseService = require './base'
+
+# IMPORTANT:
+# THIS FEATURE IS FOR INTERNAL USE ONLY.
+# MIGHT BE REMOVED IN THE FUTURE.
+# PLEASE DON'T USE THIS ENDPOINT.
```

Comment:
This doesn't make much sense. If it says "don't use it", why do we need to have it? :)

Nevertheless, having this available when it's not officially documented by the API it's not really nice. I see 3 options here:
- you don't use the SDK and make the requests by yourself, the implementation is pretty simple
- you use the new SDK (alpha version), which allows you to register services by yourself (which is exactly this use case). The new version is still not fully-featured yet though, so you have to check if it's enough what there is available
- add this service here (less preferred option)

In any case, tests are missing for this PR.